Graphs/TreeGraph/index.js

```diff
@@ -394,7 +402,8 @@ class TreeGraph extends AbstractGraph {
         } = this.getConfiguredProperties();
 
         const {
-            commonEN
+            commonEN,
```

Comment:
I don't think we use this lets remove

Reply:
No, actually we are using when node is the only image and when we hover we show name and description accordingly.